samples/cpp/tutorial_code/ImgProc/Pyramids.cpp

```diff
@@ -39,15 +39,14 @@ int main( void )
   dst = tmp;
 
   //![create_window]
-  namedWindow( window_name, WINDOW_AUTOSIZE );
   imshow( window_name, dst );
   //![create_window]
 
   //![infinite_loop]
   for(;;)
   {
     int c;
-    c = waitKey(10);
+    c = waitKey(0);
 
     if( (char)c == 27 )
```

Comment:
let me leave it as is for now. because you did proper change with your PR. i will try to solve conflicts if your PR will be merged first.